internal/logging/logging.go

```diff
@@ -255,16 +255,7 @@ func writeMessageDepth(depth int, level string, msg string, args ...interface{})
 }
 
 func printLogError(err error, ctx *MessageContext, msg string, args ...interface{}) {
-	errMsg := err.Error()
-	errw := err
-	for {
-		errw = errors.Unwrap(errw)
-		if errw == nil {
-			break
-		}
-		errMsg += ": " + errw.Error()
-	}
-	fmt.Fprintf(os.Stderr, "Error writing log message: %s\n", errMsg)
+	fmt.Fprintf(os.Stderr, "Error writing log message: %s\n", errs.JoinMessage(err))
```

Comment:
Very elegant! This feels like one of those moments in tetris where you prepared everything for that 1x4 piece and finally got it.

Reply:
Lol! To me it felt more like "Why did I do it this way the first time around?!?!" :p